### ISSUE TITLE
Uncaught exception in lightcurve plot

### ISSUE THREAD

**Issue by stefanv** [skyportal/skyportal#2668]
```
ERROR:tornado.application:Uncaught exception GET /api/internal/plot/photometry/ZTF21acaqdee?width=800&device=browser (78.197.174.32)
Traceback (most recent call last):
  File "/skyportal/skyportal/plot.py", line 331, in get_effective_wavelength
    bandpass = snc.get_bandpass(bandpass_name)
  File "/skyportal_env/lib/python3.8/site-packages/sncosmo/bandpasses.py", line 27, in get_bandpass
    return _BANDPASSES.retrieve(name)
  File "/skyportal_env/lib/python3.8/site-packages/sncosmo/_registry.py", line 160, in retrieve
    self._instances[key] = func(*args, name=name)
  File "/skyportal_env/lib/python3.8/site-packages/sncosmo/builtins.py", line 75, in load_bandpass_remote_aa
    return read_bandpass(abspath, wave_unit=u.AA,
  File "/skyportal_env/lib/python3.8/site-packages/sncosmo/bandpasses.py", line 71, in read_bandpass
    return Bandpass(t['wave'], t['trans'], wave_unit=wave_unit,
  File "/skyportal_env/lib/python3.8/site-packages/sncosmo/bandpasses.py", line 201, in __init__
    s = slice_exclude_below(trans, np.max(trans) * trim_level, grow=1)
  File "<__array_function__ internals>", line 5, in amax
  File "/skyportal_env/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 2754, in amax
    return _wrapreduction(a, np.maximum, 'max', axis, None, out,
  File "/skyportal_env/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 86, in _wrapreduction
    return ufunc.reduce(obj, axis, dtype, out, **passkwargs)
ValueError: zero-size array to reduction operation maximum which has no identity
```

**Comment by guynir42** [comment]
Closed by #2675 (I hope)

**Comment by guynir42** [comment]
Closed unless this bug comes back...
